network/common/src/main/java/org/apache/spark/network/server/StreamManager.java

```diff
@@ -47,6 +47,14 @@
   public abstract ManagedBuffer getChunk(long streamId, int chunkIndex);
 
   /**
+   * Called in response to a stream() request. The returned data is streamed to the client
+   * through a single TCP connection.
+   */
+  public ManagedBuffer openStream(String streamId) {
```

Comment:
I actually don't like how the other method (`getChunk`) uses "stream" when it doesn't really stream anything... I dislike "unframedStream" because "unframed" is irrelevant to the user, it's just being there as a contrived way to differentiate the two APIs.

I can make the javadoc more explicit about the lack of relationship with the other method. Maybe `getStreamBuffer`, although that has the same possibility of confusion with the other method. I do like `openStream` better than any other variant, though.